frontend/src/scenes/retention/retentionModalLogic.ts

```diff
@@ -43,6 +46,36 @@ export const retentionModalLogic = kea<retentionModalLogicType>([
                 return aggregationLabel(aggregation_group_type_index)
             },
         ],
+        personsQuery: [
+            (s) => [s.querySource, s.selectedInterval],
+            (querySource: RetentionQuery, selectedInterval): PersonsQuery | null => {
+                if (!querySource) {
+                    return null
+                }
+                return retentionToActorsQuery(querySource, selectedInterval ?? 0)
+            },
+        ],
+        exploreUrl: [
+            (s) => [s.personsQuery],
+            (personsQuery): string | null => {
+                if (!personsQuery) {
+                    return null
+                }
+                const query: DataTableNode = {
+                    kind: NodeKind.DataTableNode,
+                    source: personsQuery,
+                    full: true,
+                }
+                if (
+                    isInsightPersonsQuery(personsQuery.source) &&
+                    isRetentionQuery(personsQuery.source.source) &&
+                    personsQuery.source.source.aggregation_group_type_index !== undefined
```

Comment:
This can probably be swapped out for `isQueryForGroup()`, however why are you actually hiding the property filter here? Users should be able to filter the results further...

Reply:
As far as I saw it, the filter is geared towards persons, and if we have groups it's not applicable, hence this was my idea to remove it.

Reply:
Right, makes sense.